web/war/pom.xml

```diff
@@ -206,5 +206,230 @@
             </plugins>
             </build>
         </profile>
+        <profile>
+            <id>web-war-with-ui-plugins</id>
```

Comment:
Change profile ID to `with-ui-plugins`; also -- is this all the UI plugins, a basic set, specifically for the demo container?